layouts/partials/head.html

```diff
@@ -27,19 +27,23 @@
-  <script data-jsd-embedded data-key="ddb2b60b-2cc3-47f0-a434-3a6dae10166a" data-base-url="https://jsd-widget.atlassian.com" src="https://jsd-widget.atlassian.com/assets/embed.js"></script>
-  <script src="https://identity.netlify.com/v1/netlify-identity-widget.js"></script>
+	<link href="/css/bootstrap.min.css" rel="stylesheet">
+	<link href="/css/style.css?v1.0" rel="stylesheet">
+		<meta name="go-import" content="storj.io/storj git https://github.com/storj/storj">
+	<script> !function(){varanalytics=window.analytics=window.analytics||[];if(!analytics.initialize)if(analytics.invoked)window.console&&console.error&&console.error("Segment snippet included twice.");else{analytics.invoked=!0;analytics.methods=["trackSubmit","trackClick","trackLink","trackForm","pageview","identify","reset","group","track","ready","alias","debug","page","once","off","on"];analytics.factory=function(t){returnfunction(){vare=Array.prototype.slice.call(arguments);e.unshift(t);analytics.push(e);returnanalytics}};for(vart=0;t<analytics.methods.length;t++){vare=analytics.methods[t];analytics[e]=analytics.factory(e)}analytics.load=function(t){vare=document.createElement("script");e.type="text/javascript";e.async=!0;e.src=("https:"===document.location.protocol?"https://":"http://")+"cdn.segment.com/analytics.js/v1/"+t+"/analytics.min.js";varn=document.getElementsByTagName("script")[0];n.parentNode.insertBefore(e,n)};analytics.SNIPPET_VERSION="4.0.0"; analytics.load("JHfKYYbErCZRf2QFQTQNTyP0IF6RMSUC"); analytics.page(); }}();</script>
+	<script data-jsd-embedded data-key="ddb2b60b-2cc3-47f0-a434-3a6dae10166a" data-base-url="https://jsd-widget.atlassian.com" src="https://jsd-widget.atlassian.com/assets/embed.js"></script>
+	<script src="https://identity.netlify.com/v1/netlify-identity-widget.js"></script>
 	<script src="//rum-static.pingdom.net/pa-5c7947bd9a3f8300160008c5.js" async></script>
 	<link href="https://fonts.googleapis.com/css?family=Poppins:400,600,700|Roboto:300,700" rel="stylesheet">
-	<script type="text/javascript" id="inspectletjs">
-		(function() {
-			window.__insp = window.__insp || [];
-			__insp.push(['wid', 1154029405]);
-			var ldinsp = function(){ if(typeof window.__inspld != "undefined") return; window.__inspld = 1; var insp = document.createElement('script'); insp.type = 'text/javascript'; insp.async = true; insp.id = "inspsync"; insp.src = ('https:' == document.location.protocol ? 'https' : 'http') + '://cdn.inspectlet.com/inspectlet.js?wid=1154029405&r=' + Math.floor(new Date().getTime()/3600000); var x = document.getElementsByTagName('script')[0]; x.parentNode.insertBefore(insp, x); };
-			setTimeout(ldinsp, 0);
-		})();
-		</script>
+<!-- Begin Inspectlet Asynchronous Code -->
```

Comment:
We should be setting up inspeclet through segment.io. You should be able to change the inspeclet id in the segment.io dashboard to connect to the correct one

Reply:
Sounds good. I've removed the outdated tags from the head partial and have requested the WID number to update in segment from @tikhbana in https://storjlabs.atlassian.net/projects/WD/queues/custom/42/WD-23.

Note: This merge also updates the canary file per @super3 and addresses the issues with front-matter keys in the recent copy update for the community hero-nav.